selenium/che-selenium-test/pom.xml

```diff
@@ -106,11 +106,21 @@
             <artifactId>testng</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.eclipse.che.core</groupId>
+            <artifactId>che-core-api-user-shared</artifactId>
```

Comment:
I added it once to prevent NoClassDefFound exception at test executions. But I can't reproduce it anymore, so I'll remove it